/*
* Author: Manuel Olguín Muñoz <manuel@olguin.se>
* 
* Copyright© 2019 Manuel Olguín Muñoz
* See LICENSE file included in the root directory of this project for licensing and copyright details.
*/

#ifndef MINISYNCPP_NODE_H
#define MINISYNCPP_NODE_H

namespace MiniSync
{
    class Node {

    };

    class ReferenceNode
    {


    };
}

#endif //MINISYNCPP_NODE_H
